src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -84,7 +84,7 @@
     "browser.name": PREFIX + 216,
     "os.name": PREFIX + 217,
     "satisfaction": PREFIX + 218,
-    "ok": PREFIX + 219,
+    "ok": PREFIX + 219,  # now also used by release health
```

Comment:
i was assuming we'd extract this extra data under `exited`, no?

Reply:
We emit the user ID with the `ok` tag not just for exited sessions, but for any session update that is not init, but still healthy (https://github.com/getsentry/relay/blob/243e85292d785a1010256eb6b2e1748191ec6708/relay-server/src/metrics_extraction/sessions.rs#L205-L218).

The reason for this is that not all sessions exit formally AFAIK, but we still want to capture the user ID when they don't.